.gitignore

```diff
@@ -4,4 +4,4 @@ bbg-mas-x64
 bbg-darwin-x64
 bbg-win32-x64
 .vs
-dist
+dist
```

Comment:
It would be better if a new line (`\n`) was added at the end of the file.

Reply:
Resolved in [5ebb156](https://github.com/baiyang-lzy/bbg/commit/5ebb156fba0a917a701e682e6b0c00aabb3193a7).